src/jobs/qa-automation-suite.yaml

```diff
@@ -0,0 +1,48 @@
+parameters:
+  test_path:
+    description: "The path to the test suite to run"
+    type: string
+  project_name:
```

Comment:
Do we have a fixed number of option I suppose, can we list them?

Reply:
Not sure. @medleycodacy @jonathancarvalheiro any idea?

Reply:
@andreaTP Could not be fixed number, since you can create your specific project on RP. But we can suggest or have a default :qa-automation-tests

Reply:
so this parameter should be called possibly:
`report_portal_project_name` correct?
@jonathancarvalheiro what happens if we set a name that is not "registered" already in RP?
Do we have a little description on how to "create a project" in RP?

Reply:
I'll leave the description in here.
Create a project on RP. 
1. Go to the RP and login with the superadmin user (check Bitwarden for that).
2. Click on the right top corner in the user options, and select Administrate.
3. Click on the Add New Project button, and define the project name. 

Something like this, @andreaTP ?

Reply:
Could we add that to the docs of the qa-automation repo?
And add a link to there in this description?

Reply:
yes @jonathancarvalheiro but I think @DReigada suggestion stands. please.

Reply:
I'll do that.

Reply:
Created PR to handbook: https://github.com/codacy/handbook/pull/16/files

Reply:
Link: https://handbook.dev.codacy.org/product/engineering/tools/reportportal.html